applications/registrar/src/reg_authn_req.erl

```diff
@@ -270,82 +269,22 @@ get_auth_user_in_account(Username, Realm, AccountDB) ->
 %%-----------------------------------------------------------------------------
 %% @private
 %% @doc
-%% lookup auth by IP in cache/database and return the result
-%% @end
-%%-----------------------------------------------------------------------------
--spec lookup_account_by_ip(ne_binary()) ->
-                                  {'ok', kz_proplist()} |
-                                  {'error', 'not_founnd'}.
-lookup_account_by_ip(IP) ->
-    lager:debug("looking up IP: ~s in db ~s", [IP, ?KZ_SIP_DB]),
-    kapps_util:get_ccvs_by_ip(IP).
-
-%%-----------------------------------------------------------------------------
-%% @private
-%% @doc
 %%
 %% @end
 %%-----------------------------------------------------------------------------
 -spec check_auth_user(kz_json:object(), ne_binary(), ne_binary(), kz_json:object()) ->
                              {'ok', auth_user()} |
                              {'error', any()}.
```

Comment:
While you're at it can you specialize to `{error, disabled}`?
Also, `maybe_auth_method` can only return `{ok, auth_user()}`. We should update that too maybe.